Makefile

```diff
@@ -0,0 +1,36 @@
+IXO_ROOT		   ?= https://github.com/ixofoundation/ixo-blockchain
+IXO_RELEASE	   ?= v1.6.0
+IXO_DST_FOLDER ?= ixo-blockchain-${IXO_RELEASE}
+GOOS 					 ?= linux
+GOARCH				 ?= amd64
+
+all: clone build-ixod
+
+clone:
+	@echo "Cloning repo ${IXO_ROOT}"
+	test -d ${IXO_DST_FOLDER} || \
+	git clone ${IXO_ROOT} ${IXO_DST_FOLDER}
+	cd ${IXO_DST_FOLDER} && \
+	git fetch --all && \
+	git checkout ${IXO_RELEASE}
+
+build-ixod: clone
+	docker run \
+	-v "$(CURDIR)/${IXO_DST_FOLDER}:/app" \
+	-v "$(CURDIR)/bin:/go/bin/" \
+	-e GOARCH=${GOARCH} \
+	-e GOOS=${GOOS} \
+	--workdir "/app" golang:1.16 make install
+
+build-image: clone
+ifndef network
+	$(error network is undefined)
```

Comment:
add check for IXO_RELEASE var?

Reply:
Why do I need this?
IXO_RELEASE is defined at the top of the Makefile.